python/tests/test_aac/context/test_source_location.py

```diff
@@ -5,5 +5,9 @@
 class TestSourceLocation(TestCase):
     def test_source_location(self):
         loc = SourceLocation(1, 2, 3, 4)
-        
-        self.assertTrue(isinstance(loc.to_tuple(), tuple))
+
+        self.assertIsInstance(loc.to_tuple(), tuple)
```

Comment:
I would expand this test to check the mapping of the tuple entries as well. So, the `SourceLocation` class has each entry of the tuple designated as line, column, position, and span. I would do a check for the mapping results.

Reply:
@bunchw don't forget to check the source location mapping of the tuple entries.